content/minfraud/_api-documentation/request-object-reference.mdx

```diff
@@ -0,0 +1,962 @@
+  >
+    The total order amount for the transaction before taxes and discounts.
+  </Property>
+
+  <Property
+    example="USD"
+    name="currency"
+    tags={{
+      'Max Length': 3,
+    }}
+    type="string"
+  >
+    The [ISO 4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) for the
+    currency used in the transaction.
+  </Property>
+
+  <Property
+    example="FIRST"
+    name="discount_code"
+    tags={{
+      'Max Length': 255,
+    }}
+    type="string"
+  >
+    The discount code applied to the transaction. If multiple discount codes
+    were used, please separate them with a comma.
+  </Property>
+
+  <Property
+    example="af12"
+    name="affiliate_id"
+    tags={{
+      'Max Length': 255,
+    }}
+    type="string"
+  >
+    The ID of the affiliate where the order is coming from. No specific format
+    is required.
+  </Property>
+
+  <Property
+    example="saf42"
+    name="subaffiliate_id"
+    tags={{
+      'Max Length': 255,
+    }}
+    type="string"
+  >
+    The ID of the sub-affiliate where the order is coming from. No specific
+    format is required.
+  </Property>
+
+  <Property
+    example="https://www.google.com/"
+    name="referrer_uri"
+    tags={{
+      'Max Length': 1024,
+    }}
+    type="string"
+  >
+    The URI of the referring site for this order. Needs to be absolute and have
+    a URI scheme such as `https://`.
+  </Property>
+
+  <Property
+    example="true"
+    name="is_gift"
+    type="boolean"
+  >
+    Whether order was marked as a gift by the purchaser.
+  </Property>
+
+  <Property
+    example="false"
+    name="has_gift_message"
+    type="boolean"
+  >
+    Whether the purchaser included a gift message.
+  </Property>
+</Schema>
+
+<Schema
+  name="Request | Shopping Cart"
+  type="array<object>"
+>
+  This is an _array_ of shopping cart item objects. A shopping cart should
+  consist of an array of one or more item objects.
+</Schema>
+
+<Schema
+  name="Request | Shopping Cart | Item"
+  services="*"
+>
+  <Property
+    name="category"
+    tags={{
+      'Max Length': 255,
+    }}
+    type="string"
+  >
+    The category of the item. This can also be a hashed value; see below.
+  </Property>
+
+  <Property
+    name="item_id"
+    tags={{
+      'Max Length': 255,
+    }}
+    type="string"
+  >
+    Your internal ID for the item. No specific format is required. This can also
+    be a hashed value; see below.
+  </Property>
+
+  <Property
+    name="item_id"
+    tags={{
+      'Min': 0,
+      'Max': '10e13-1',
+    }}
+    type="integer"
+  >
+    The quantity of the item in the shopping cart. The quantity of the item in
+    the shopping cart. The value must be a whole number.
+  </Property>
+
+  <Property
+    name="price"
+    tags={{
+      'Min': 0,
+      'Max': '1e14-1',
+    }}
+    type="decimal"
+  >
+    The per-unit price of this item in the shopping cart. This should use the
+    same currency as the order currency.
+  </Property>
+
+  <Footnote>
```

Comment:
`<Footnote>` is isn't rendering properly:

![image](https://user-images.githubusercontent.com/1154185/108571325-ef1e2080-72dd-11eb-9213-ba8dde80088f.png)

Reply:
I pushed a fix for this.